src/components/opt-in-out/index.tsx

```diff
@@ -0,0 +1,76 @@
+import { useState } from 'react'
+import { useRouter } from 'next/router'
+import Cookies from 'js-cookie'
+import { IconSignOut16 } from '@hashicorp/flight-icons/svg-react/sign-out-16'
+import '@reach/dialog/styles.css'
+import Button from 'components/button'
+import OptOutForm from './components/opt-out-form'
+import Dialog from 'components/dialog'
+import { getLearnRedirectPath } from './helpers/get-learn-redirect-path'
+import { PlatformOptionsType, OptInOutProps } from './types'
+
+// Could these go in the config? or I could source the base urls elsewhere
+export const PLATFORM_OPTIONS: PlatformOptionsType = {
+  learn: {
+    base_url: 'https://learn-git-ksspike-opt-in-redirects-hashicorp.vercel.app', // FOR TESTING PURPOSES NEED TO UPDATE for - 'https://learn.hashicorp.com/'
+    getRedirectPath: getLearnRedirectPath,
+    cookieKey: 'learn-beta-opt-in',
+  },
+  'waypoint-io': {
```

Comment:
⛏️ Nitpick thought - Should these keys match the `platformOptions` enum in `opt-out-form`?

Or maybe `opt-out-form`'s `platformOptions` enum should match these keys? (slash should match `OptInPlatformOption` from the `_middleware` file)